pipeline-model-definition/src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/BasicModelDefTest.java

```diff
@@ -78,7 +78,7 @@ public static void setUpAgent() throws Exception {
 
     @Issue("JENKINS-47363")
     // Give this a longer timeout
-    @Test(timeout=5 * 60 * 1000)
+    @Test(timeout=10 * 60 * 1000)
```

Comment:
And now it is [passing in 98s](https://ci.jenkins.io/job/Plugins/job/pipeline-model-definition-plugin/job/PR-762/9/testReport/org.jenkinsci.plugins.pipeline.modeldefinition/BasicModelDefTest/stages300/). :shrug: